SlackBotPrototype/API/MessageParser.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Console = System.Console;
+using edu.stanford.nlp.ling;
+using edu.stanford.nlp.tagger.maxent;
+
+namespace API
+{
+	public class MessageParser : IMessageParser
+	{
+		private readonly string _modelsDirectory;
+		
+		public MessageParser(string nlpFolder)
+		{
+			_modelsDirectory = nlpFolder + @"\models";
+		}
+
+		public string HandleRequest(string message)
+		{
+			var wordSet = BuildWordSet(message);
+			var weatherProvider = new DarkSkyWeatherProvider(ConfigConstants.DarkSkyApiSecret);
+			
+			if (wordSet.Contains("wear", StringComparer.InvariantCultureIgnoreCase) &&

```

Comment:
Agreed, would definitely have made deployment easier too! I think there's a lot of room for improving how it's used, we could at least check the part of speech. I think another way to expand it is to actually parse the sentence into a tree and use a `tregex` to match specific sentence patterns (see https://nlp.stanford.edu/nlp/javadoc/javanlp/edu/stanford/nlp/trees/tregex/TregexPattern.html).